src/machines/java/com/enderio/machines/common/blockentity/base/MachineBlockEntity.java

```diff
@@ -166,7 +182,7 @@ protected boolean supportsIOMode(Direction side, IOMode mode) {
     }
 
     @Override
-    public ModelData getModelData() {
+    public @NotNull ModelData getModelData() {
```

Comment:
Sorry, but I'm not really sure I understand what you mean since my understanding of the annotation system isn't good enough. Do you mean that I should remove the "@NotNull"? The reason that I inserted it was that befor I had it Intellij threw a warning at me saying that "Not annotated method overrides method annotated with @NotNull". It seems to be done that same way in the class "XPVacuumBlockEntity" line 88 in the method "getCapability". What's the difference between the two cases so I understand for next time?

Reply:
it is not wanted there either.
this file has the annotations for a whole package and that replaces the annotations on each field
<https://github.com/SleepyTrousers/EnderIO-Rewrite/blob/dev/1.19.x/src/main/java/com/enderio/package-info.java>